### PR TITLE
Lisflood merge Epic + waterbalance updates

### PR DESCRIPTION
**Pull request by doc78**

### CODE REVIEW

File: src/lisflood/hydrological_modules/soilloop.py
Comment (StefaniaGrimaldi) [reply src/lisflood/hydrological_modules/soilloop.py:495]: The lines 493-1005 are the soilloop of LISFLOODV3.1  and they include a bug fix in the option "simulatePF". I think we can remove those lines. I will explain in the documentation of the merge also the bug fix in the the option "simulatePF"

File: src/lisflood/Lisflood_initial.py
Comment (StefaniaGrimaldi) [inline src/lisflood/Lisflood_initial.py:16]: I think we should keep lines 1-16 "Copyright 2019 European Union": it was my mistake to delete them from this module.

File: src/lisflood/hydrological_modules/groundwater.py
Comment (StefaniaGrimaldi) [inline src/lisflood/hydrological_modules/groundwater.py:111]: Can we remove "### stef" before the merge?

File: src/lisflood/hydrological_modules/indicatorcalc.py
Comment (StefaniaGrimaldi) [inline src/lisflood/hydrological_modules/indicatorcalc.py:98]: Can we remove ############### Sept 17?

File: src/lisflood/hydrological_modules/miscInitial.py
Comment (StefaniaGrimaldi) [reply src/lisflood/hydrological_modules/miscInitial.py:132]: OK! Can we also remove ## C, Stef in line 133?

File: src/lisflood/hydrological_modules/soilloop_merge2.py
Comment (StefaniaGrimaldi) [inline src/lisflood/hydrological_modules/soilloop_merge2.py:1]: The following scripts:
- soilloop_merge2.py
- soilloop_imposeMinLoops.py
- soilloop_alwaysMaxNoLoops.py
- soilloop_OLDvarstruct.py
are NOT required in the master branch.

The scripts listed above allow to keep track of the steps and of the investigations that were done to edit the soilloop. They might be useful in the future for other analysis but at this stage they are not required to run the code.

File: src/settings_tpl.xml
Comment (doc78) [reply src/settings_tpl.xml:1]: Ok, maybe we can keep only "lisfloodSettings_reference.xml" and change the name of the settings template file used in the Docker file

File: src/lisflood/__init__.py
Comment (StefaniaGrimaldi) [inline src/lisflood/__init__.py:13]: I think we should change the date here before the release

File: tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12/inflow_6h/chanqWin.tss
Comment (StefaniaGrimaldi) [inline tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12/inflow_6h/chanqWin.tss:1]: We can delete the folder "tests/data/LF_ETRS89_UseCase/**reference_mergeMinLoops12**" before the push

File: tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12/inflow_6h/dis.tss
Comment (StefaniaGrimaldi) [inline tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12/inflow_6h/dis.tss:1]: We can delete the folder "tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12" before the push

File: tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12/inflow_6h/inflow.tss
Comment (StefaniaGrimaldi) [inline tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12/inflow_6h/inflow.tss:1]: We can delete the folder "tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12" before the push

File: tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12/inflow_daily/chanqWin.tss
Comment (StefaniaGrimaldi) [inline tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12/inflow_daily/chanqWin.tss:1]: We can delete the folder "tests/data/LF_ETRS89_UseCase/reference_mergeMinLoops12" before the push

File: tests/data/LF_lat_lon_UseCase/reference_ricewateruseOFF/chanqWin.tss
Comment (StefaniaGrimaldi) [inline tests/data/LF_lat_lon_UseCase/reference_ricewateruseOFF/chanqWin.tss]: The folder tests/data/LF_lat_lon_UseCase/**reference_ricewateruseOFF**/ can be deleted before the push.

File: src/settings_tpl.xml
Comment (StefaniaGrimaldi) [reply src/settings_tpl.xml:1]: I think it is perfect, thanks!

File: src/lisflood/Lisflood_initial.py
Comment (doc78) [reply src/lisflood/Lisflood_initial.py:16]: Ok, I resumed the lines. Maybe we should update the year too, in all the files where the Copyright is written.

File: src/lisflood/__init__.py
Comment (StefaniaGrimaldi) [inline src/lisflood/__init__.py:16]: We should add Damien Decremer and Corentin Carton De Wiart. For now, Lorenzo Mentaschi is not able to work as a maintainer.

**Review by corentincarton — APPROVED** [review]

File: requirements.txt
Comment (doc78) [reply requirements.txt]: I have already prepared the requirements.txt with >= instead of ==, now we can apply this change since now lisflood-utilities is fixed and also the date fix are applied here (were giving issues otherwise). Environment.yml will be updated with the final environment used after all the changes, as well as the docker file. 
If I'm not wrong, pcraster is not needed as a pip package (it is installed as a conda-forge package).

**Review by ecCinziaMazzetti — APPROVED** [review]

**Pull request merged**
